.github/workflows/checkAndSubmitAddonMetadata.yml

```diff
@@ -225,11 +227,69 @@ jobs:
       uses: peter-evans/close-issue@v3
       with:
         issue-number: ${{ inputs.issueNumber }}
-  codeQL-analysis:
+
+  virusTotal-analysis:
     needs: createPullRequest
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [ 3.11 ]
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write
+    env:
+      API_KEY: ${{ secrets.virusTotalApiKey }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Download add-on metadata
+        uses: actions/download-artifact@v4
+        with:
+          name: addonMetadata
+      - name: Install virusTotal
+        run: choco install vt-cli
+      - name: Set Virus Total analysis status
+        id: setVirusTotalAnalysisStatus
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const setVirusTotalAnalysisStatus = require('./.github/workflows/virusTotalAnalysis.js')
+            setVirusTotalAnalysisStatus({core})
+      - name: Upload results
+        id: uploadResults
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: VirusTotal
+          path: vt.json
+          overwrite: true
+      - name: Create pull request
+        id: cpr
+        if: failure()
+        uses: peter-evans/create-pull-request@v6
+        with:
+          add-paths: falsePositiveAddons.json
+          title: Add false positive add-on (${{ steps.setVirusTotalAnalysisStatus.outputs.addonId }})
+          branch: falsePositiveAddon${{ github.event.issue.number }}
+          commit-message: Add false positive add-on (${{ steps.setVirusTotalAnalysisStatus.outputs.addonId }})
+          body: "This add-on needs to be reviewed by NV Access due to VirusTotal analysis failure"
+          author: github-actions <github-actions@github.com>
+          delete-branch: true
+      - name: Warn if analysis fails
+        if: failure()
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ inputs.issueNumber }}
+          body: |
+            VirusTotal has flagged this add-on as malicious.
+            You can open this link and [download artifacts containing the results of the analysis](${{ steps.uploadResults.outputs.artifact-url }}).
+            If you think this is a false positive or want more details, contact NV Access.
+  codeQL-analysis:
```

Comment:
Why does codeql need virus total?

Can we make the PR to override a separate action, that happens when codeql or virus total fails? 
I think we should use one manual approval method for both failures.

Reply:
I¡ll accept your suggestion. I thought that CodeQL consumes many minutes and that, if VirusTotal fails, it may be skipped.